crux-lucene/src/crux/lucene.clj

```diff
@@ -116,13 +116,15 @@
         (let [index-searcher (IndexSearcher. directory-reader)
               q (TermQuery. (Term. "meta" "latest-completed-tx"))
               d ^ScoreDoc (first (.-scoreDocs (.search index-searcher q 1)))]
-          (when d (.get (.doc index-searcher (.-doc d)) "latest-completed-tx")))))))
+          (when d (some-> (.doc index-searcher (.-doc d))
+                          (.get "latest-completed-tx")
+                          (Long/parseLong))))))))
```

Comment:
newline - I'm fine with the block of a `when` being on the same line as the condition when it's only one line, but when the block goes on to the next line, let's drop it down

```suggestion
          (when d 
            (some-> (.doc index-searcher (.-doc d))
                    (.get "latest-completed-tx")
                    (Long/parseLong))))))))
```